ten-simple-rules-dockerfiles.Rmd

```diff
@@ -420,6 +428,7 @@ _Listing&nbsp;3_: Example labels.
 
 It is often helpful to provide usage instructions, i.e., how to `docker build` and `docker run` the image, _within_ the `Dockerfile`, either at the top or bottom where the reader is likely to find it.
 Such documentation is especially relevant if volume mounts, specific names, or ports are important for using the container, see for example the final lines of Listing&nbsp;1.
+These instructions are not limited to `docker <command>`, but include usage of bespoke scripts, `Makefile`s, or `docker-compose` (see \ruleref{rule:interactive} and \ruleref{rule:usage}).
```

Comment:
I've sensed that "bespoke" is a popular word :) I think it's overused a bit, maybe take it out here, or choose a different term. I wonder how many times we use it in the paper? Or maybe it just sticks out to me because I didn't use / see it a lot before this paper? :thinking:

Reply:
We use it :two: times :-). I admit I like it.

Reply:
ahh okay that's not very many, I'm ok with that :)